py/mockbuild/buildroot.py

```diff
@@ -161,6 +161,13 @@ def _init(self, prebuild, do_log):
                 self.state.start(update_state)
                 self.pkg_manager.update()
                 self.state.finish(update_state)
+        else:
+            self._fixup_build_user()
```

Comment:
What will that achieve? The purpose of _fixup_build_user() is (only) to modify the /etc/passwd file in the mock root, and as far as I can see dropPrivsTemp() does nothing like that?